.github/workflows/gcc-debug-proof-producer-nix-check.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and Test on Linux Platforms
+
+on:
+  workflow_call:
+
+jobs:
+  build-and-test:
+    name: "Build and test Linux with gcc"
+    runs-on: [self-hosted, Linux, X64, aws_autoscaling]
+    steps:
+      # https://github.com/actions/checkout/issues/1552
+      - name: Clean up after previous checkout
+        run: chmod +w -R ${GITHUB_WORKSPACE}; rm -rf ${GITHUB_WORKSPACE}/*;
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Run checks for debug proof-producer
```

Comment:
Maybe highlight somehow that no actual tests are running here, just basic compiled binary check?

Reply:
True, thanks for noticing

Reply:
corrected